README.md

```diff
@@ -1,135 +1,268 @@
-# Format Analysis Workbench (FAW)
+# Introduction
 
 <img src="./docs/logo/original.png" width="300px" align="right"/>
 
-The Format Analysis Workbench (FAW) is a platform for running and analyzing the
-output from any number of parsers dealing with a single file format. It is meant
-to be a workbench for developing tools which aid in understanding the de facto
-formats which naturally emerge from open standards.
-
-<br clear="right" />
-
-## Example Screenshots
+As a data format gains popularity, the number of parsers and parser-like
+tools developed to process it increases significantly. Even in the
+presence of a format specification, these tools often interpret it in
+subtly different ways. Moreover the data format and its specification
+also tend to evolve over time to accommodate new use cases and user
+demands. Together, this can cause significant divergence in the output
+of parsers leading to *parser differentials*, a situation where two
+parsers exhibit semantically different behavior for the same input,
+which and is a well-known and practically exploited vulnerability.
+
+Identifying parser differentials and exploits is often a time-consuming
+task involving multiple iterations of hypothesizing, developing
+special-purpose tools, running against a file corpora and analysing the
+results. Reducing the manual aspects of this cycle can serve to
+significantly accelarate this loop and enable investigations on much
+larger file corpora and with more parsers and tools than manually
+possible. Further, such scaling also enables the discovery of
+statistical correlations between the outputs of the tools which can help
+to surface the root causes of many differentials and exploits.
+
+The Feature Analysis Workbench (FAW) provides a workbench for
```

Comment:
Should this be the 'Format Analysis Workbench' or are we changing the name? (If so, we'll need to update the title on the logo as well.)

Reply:
Oops! I have fixed it.